.github/workflows/maven.yml

```diff
@@ -28,13 +28,17 @@ jobs:
         continue-on-error: true
       - name: Build with Maven (Windows)
         env:
-          AWS_REGION: us-west-2
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
For GG we cannot have this. Our tests which run in github may not access AWS resources.

Fine for you, for now.